packages/desktop-client/src/components/reports/graphs/NetWorthGraph.tsx

```diff
@@ -74,6 +75,7 @@ export function NetWorthGraph({
   };
 
   const off = gradientOffset();
+  const gradientId = `splitColor-${id}`;
```

Comment:
An alternative would be to use `splitColor-${off}` since the conflict is fine then. To me using an id feels cleaner/safer to avoid further conflicts if different styling is applied to the gradient depending on other params.